client/components/Editor/schemas/image.ts

```diff
@@ -103,7 +118,7 @@ export default {
 						renderHtmlChildren(isReact, caption, 'div'),
 					]),
 				],
-			] as DOMOutputSpec;
+			] as unknown) as DOMOutputSpec;
```

Comment:
I have a hunch (and it's really just that) that if you define `imgTag` as follows:

```ts
const imgTag = [...] as const;
```

you may not need this `unknown` cast. Doing this promises Typescript that you won't mutate that array and will narrow down the type considerably to the point where it may be compatible with `DOMOutoutSpec`.